### PR TITLE
update copyright year on the beginning of files to 2021, also refacto…

### PR DESCRIPTION
**Pull request by notseanyay**
…red some modules like liquidfiller and xray

### CODE REVIEW

File: src/main/java/minegame159/meteorclient/commands/CommandManager.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public static void init() {
         addCommand(new SwarmCloseConnections());
         addCommand(new SwarmEscape());
         addCommand(new SwarmFollow());
-        addCommand(new SwarmGoto());
+        addCommand(new SwamGoTo());
```
Comment (seasnail8169) [inline src/main/java/minegame159/meteorclient/commands/CommandManager.java:38]: this is a spelling mistake please fix it

File: src/main/java/minegame159/meteorclient/commands/commands/swarm/SwamGoTo.java [hunk 2]
```diff
@@ -11,9 +16,9 @@
 
 import static com.mojang.brigadier.Command.SINGLE_SUCCESS;
 
-public class SwarmGoto extends Command {
+public class SwamGoTo extends Command {
```
Comment (seasnail8169) [inline src/main/java/minegame159/meteorclient/commands/commands/swarm/SwamGoTo.java:19]: again this is a mistake i assume

**Comment by seasnail8169** [comment]
the spelling error "swam" is still there

**Comment by notseanyay** [comment]
Wtf you mean spelling error "swarm" i just changed the capitalization of the "goto"

**Comment by seasnail8169** [comment]
Had to manually resolve the conflicts, the commits were added but the pull request not

**Pull request closed without merging**